docs/run_status.md

```diff
@@ -0,0 +1,124 @@
+# Run Status tracking in Kedro-Viz
+
+The Run Status feature in Kedro-Viz provides tracking and visualisation of your pipeline execution, giving you detailed insights into your workflow's performance and helping you quickly identify and debug issues.
+
+## Overview
+
+Run Status tracks three main aspects of your pipeline execution:
+
+1. **Pipeline execution**: Overall run status, duration, and timing
+2. **Node execution**: Individual node performance, success/failure status, and execution time
+3. **Dataset operations**: Dataset loading and saving operations with size information
+
+The feature automatically collects execution data during your `kedro run` and presents it in an intuitive visual format within the Kedro-Viz interface.
+
+## Getting started
+
+### Prerequisites
+
+- Kedro-Viz must be installed in your environment
+- Your project must be a valid Kedro project
+
+### Running a pipeline with Run Status tracking
+
+To generate run status data, simply execute your pipeline using the standard Kedro command:
+
+```bash
+kedro run
+```
+
+!!! warning
+    Run Status tracking is currently only available for full pipeline runs using `kedro run`. Partial pipeline runs (using `--from-nodes`, `--to-nodes`, `--tags`, `--pipeline`, etc.) are not supported and will not generate run status data.
+
+This will:
+- Execute your pipeline normally
```

Comment:
Some formatting issues with this table https://kedro--2436.org.readthedocs.build/projects/kedro-viz/en/2436/run_status/#running-a-pipeline-with-run-status-tracking